gremlin-go/driver/graphBinary_test.go

```diff
@@ -52,34 +53,83 @@ func readToValue(buff *bytes.Buffer) interface{} {
 func TestGraphBinaryV1(t *testing.T) {
```

Comment:
@xiazcy @L0Lmaker Have we investigated binary comparison tests? I am worried about the tests that utilize our serialization code to test serialization code because if we made a symmetric bug we will not notice it.

Reply:
I was able to add binary comp tests for big integer since it had the byte conversions refactored out into seperate funcs. We will need to add it for the other types though.

Reply:
Would it be a lot to add it in this pr? Could we find some examples online or use the python client?
Take a look here https://github.com/lyndonb-bq/tinkerpop/blob/master/gremlin-python/src/main/python/tests/structure/io/test_graphbinaryV1.py